distributed/cli/dask_scheduler.py

```diff
@@ -70,11 +72,14 @@ def del_pid_file():
         with ignoring(ImportError):
             from distributed.bokeh.scheduler import BokehScheduler
             services[('bokeh', bokeh_internal_port)] = BokehScheduler
-    scheduler = Scheduler(loop=loop, services=services)
+    scheduler = Scheduler(loop=loop, services=services,
+                          scheduler_file=scheduler_file)
     scheduler.start(addr)
 
     bokeh_proc = None
     if _bokeh:
+        if bokeh_port == 0:
+            bokeh_port = open_port()
```

Comment:
We should really let Bokeh open its port for real here. The "open port" might be reused by something else otherwise (even in another unrelated process)...